### ISSUE TITLE
UX/accessibility Issue - Cannot reach the track switches with tab on Basic Info

### ISSUE THREAD

**Issue by PereViki** [topcoder-platform/community-app#1397]
**Describe the bug**
All affordances should be available through tabbing through the site, however, track switches cannot be reached from the keyboard. 

**To Reproduce**
1. Go to 'https://www.topcoder-dev.com/settings/profile'
2. Tab through the page
3. See error

**Expected behavior**
The user should be able to set the tracks from the keyboard. 

**Screenshots**
http://www.screencast.com/t/d3XEevBg0BK

**Desktop: MacBook Pro**
 - OS: Mac OS High Sierra
 - Browser: Chrome
 - Version: 69.0.3497.92

@ThomasKranitsas

**Comment by ThomasKranitsas** [comment]
I don't think this is an issue... It's not a radio button thus it kinda makes sense to me to not being able to change the options with tab button.

**Comment by PereViki** [comment]
@ThomasKranitsas 
I am referring to accessibility issues, like when people with disability would like to browse the site with a screen reader. They would not be able to do so.

**Comment by PereViki** [comment]
WCAG 2.0 Guidelines about keyboard accessibility: https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#keyboard-operation
